edx_rest_framework_extensions/permissions.py

```diff
@@ -18,15 +16,44 @@ def has_permission(self, request, view):
         return request.user and request.user.is_superuser
 
 
+class IsStaff(BasePermission):
+    """
+    Allows access to "global" staff users..
+    """
+    def has_permission(self, request, view):
+        return request.user.is_staff
+
+
+class IsUserInUrl(BasePermission):
+    """
+    Allows access if the the requesting user matches the user in the URL.
+    """
+    def has_permission(self, request, view):
+        user_parameter_name = 'username'
+        url_username = (
+            request.GET.get(user_parameter_name, '') or
+            getattr(request, 'parser_context', {}).get('kwargs', {}).get(user_parameter_name, '')
+        )
+        return request.user.username.lower() == url_username.lower()
+
+
+class IsJwtAuthenticated(BasePermission):
+    """
+    Returns whether the request was successfully authenticated with JwtAuthentication.
+    """
+    def has_permission(self, request, view):
+        return is_jwt_authenticated(request)
+
+
 class JwtHasScope(BasePermission):
```

Comment:
I think I'd like to rename this to `HasJwtScope`.  I think starting with `Has` is more consistent with the ones that start with `Is`.

Reply:
I think with the other permissions the subject is the permission class. With this one, the subject is the JWT, hence `JwtHasScope`.

Reply:
Leaving as-is.  Thanks.